### DISCUSSION TITLE
Minimap

### DISCUSSION THREAD

**Discussion by thomastdh** [voxeltycoon/issues#929] [UI&UX]
Once you get past the starting area and you get a couple of side regions, it feels very hard to navigate good/fast to where things are located and where you are. 
Adding a small/medium minimap with resources, be it typical supcom style or popup openttd style, both could work very well and would help a lot. 

ty for listening.

**Comment by gralewski** [comment]
Navigation over map is crucial. Most players won't get back to you with feedback - just stop playing and recommending the game and you may even not know why. Please consider this QOL feature as something important.

The game is very fun and playable in current form but missing some essential features.

My use case is this - I made couple hundred millions USD and decided to buy more areas and rebuild everything. The world is very big. I want to build over whole world, but when I start moving and rotating over the map i just instantly get lost. Where is north? Am I on the side or in the center?

We need minimap and most importantly zoom out unlocking.
